parquet-tools/src/main/java/org/apache/parquet/tools/command/MergeCommand.java

```diff
@@ -41,6 +46,21 @@
           "   <output> is the destination parquet file"
   };
 
+  private static final String DEFAULT_KEY_VALUE_MERGE_STRATEGY = new DefaultKeyValueMetadataMergeStrategy().getClass().getName();
```

Comment:
You do not need to instantiate an object to get its class. Simply use `DefaultKeyValueMetadataMergeStrategy.class`

Reply:
My bad. Fixed. Thanks